src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.java

```diff
@@ -74,12 +73,15 @@
     startDatePicker = createDatePicker();
     endDatePicker = createDatePicker();
     $$$setupUI$$$();
+    startDatePicker.getSettings().setDateRangeLimits(null, LocalDate.now());
+    startDatePicker.setDateToToday();
     startDatePicker.getSettings().setGapBeforeButtonPixels(0);
     startDatePicker.getComponentDateTextField().setPreferredSize(exportDirField.getPreferredSize());
     startDatePicker.getComponentToggleCalendarButton().setPreferredSize(exportDirChooseButton.getPreferredSize());
     endDatePicker.getSettings().setGapBeforeButtonPixels(0);
     endDatePicker.getComponentDateTextField().setPreferredSize(exportDirField.getPreferredSize());
     endDatePicker.getComponentToggleCalendarButton().setPreferredSize(exportDirChooseButton.getPreferredSize());
+    endDatePicker.setEnabled(false);
```

Comment:
This changes previous behavior, where we let the user select an end date **before** a start date.

Reply:
But then we were checking for empty start date during submission anyways, so why not disable the end date itself till the user sets a start date?

Reply:
I'm ok with disabling it, but I think it's worth to mention that this changes behavior, just to be sure that we're all ok with this change... 

I don't really know if users want to set the end before the start, but it's supported on the current version. It might be safe to change it, though.